src/index.js

```diff
@@ -0,0 +1,59 @@
+import './styles.css';
+import Contact from './contact';
+import Menu from './menu';
+import spaghettiImg from './spaghetti.jpg';
+import foodImg from './food.jpg';
+import beansImg from './beans.jpg';
+
+const initialLoad = () => {
+  const headerInfo = document.createElement('div');
+  headerInfo.classList.add('row', 'header__info');
+
+  const headerTextContent = document.createElement('div');
+  headerTextContent.classList.add('col', 'header__text-content');
+
+  const headerTitle = document.createElement('h2');
+  headerTitle.classList.add('header__title');
+  headerTitle.textContent = 'Ashahs Eatery';
+
+  const headerContent = document.createElement('p');
+  headerContent.classList.add('header__content');
+  headerContent.textContent = 'Satisfying people’s hunger for life’s simple pleasures';
+
+  headerTextContent.appendChild(headerTitle);
+  headerTextContent.appendChild(headerContent);
+
+  const headerImgContent = document.createElement('div');
+  headerImgContent.classList.add('col', 'header__img-content');
+
+  headerInfo.appendChild(headerTextContent);
+
+  return headerInfo;
```

Comment:
- [ ] Here, the `initialLoad` method should be a module and in another file just like you have the content and menu content. Since the content is considered the home page content and from the requirement, it says each tabbed content should be in its own module. Here is a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) of the requirement from pathwright.